app/models/training/question.rb

```diff
@@ -14,16 +14,56 @@ def answer
       @answer ||= Answer.new(json: json)
     end
 
+    # @return [Boolean]
+    def skippable?
+      !always_show_question
+    end
+
     # @return [String] powered by JSON not type
     def to_partial_path
       partial = multi_select? ? 'check_boxes' : 'radio_buttons'
+
+      return "feedback/#{partial}" if feedback_question?
+
       partial = "learning_#{partial}" if formative_question?
       "training/questions/#{partial}"
     end
 
     # @return [Boolean]
     def multi_select?
-      confidence_question? ? false : answer.multi_select?
+      if feedback_question?
+        response_type # FIXME: this field smells
+      elsif confidence_question?
+        false
+      else
+        answer.multi_select?
+      end
+    end
+
+    # @return [Boolean] feedback free text
+    def free_text?
+      feedback_question? && options.empty?
+    end
+
+    # @return [Boolean]
+    def has_hint?
+      hint.present?
+    end
+
+    # @return [Boolean]
+    def has_other?
+      other.present?
+    end
+
+    # @param index [Integer]
+    # @return [Boolean]
+    def last_option?(index)
```

Comment:
@jack-coggin replace this ☝🏼 with [this](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/1102/commits/5d9a7535095edb29846793d49d063f235c37047e#diff-f6e88aae6cdd83c51368b10859eee7223f82b107c0672757c0c8c87f0799e19aL90-L92)

Reply:
@peterdavidhamilton I was trying to reference the last option in the list of potential answers, [this](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/1102/commits/5d9a7535095edb29846793d49d063f235c37047e#diff-f6e88aae6cdd83c51368b10859eee7223f82b107c0672757c0c8c87f0799e19aL90-L92) seems to be a way to reference the last feedback question?

Reply:
`Training::Answer::Option`

Reply:
So the way we do it can be the same; you should be able to define a `#last?` instance method for that class.